data/json/monsters/nether.json

```diff
@@ -328,7 +328,7 @@
     "id": "mon_flaming_eye",
     "type": "MONSTER",
     "name": { "str": "flaming eye" },
-    "description": "An enormous disembodied eyeball the size of a person, flying through the air through some unknown agency.  Wreathed in unnatural flickering blue flame, it possesses a blazing yellow iris with a slitted pupil like that of a cat, and trails a set of flailing black tendrils as it slowly drifts about.  Its unearthly presence fills you with dread at the prospect of falling under its baleful gaze.",
+    "description": "An enormous disembodied eyeball the size of a person, flying through the air through some unknown agency.  Wreathed in unnatural flickering blue flame, it possesses a blazing yellow iris with a slitted pupil like that of a cat, and trails a set of flailing black tendrils as it slowly drifts about.  You involuntarily get a heavy, sinking feeling deep in your gut as its gaze lazily centers itself onto you.",
```

Comment:
I’d personally suggest avoiding a direct mention of the eye gazing upon the player, as it feels a bit weird to read that the eye centres it’s vision upon you while you’re either out of its sight range, or the enemy is busy dealing with other creatures.

Reply:
changed it up a little, how is it now?

Reply:
Boomstick told me this was good on discord so I'll resolve